### DISCUSSION TITLE
Is there a way to clear log file?

### DISCUSSION THREAD

**Discussion by char990** [gabime/spdlog#3047] [General]
I am using
str_logger = spdlog::rotating_logger_mt("r_log", "/tmp/r_log", 128*1024, 4);
to create a rotating log.
And output some logs in the file(s).

But is there a way to clear/remove all file(s) and continue logging with empty "/tmp/r_log"?

There is a "truncate" argument. But it's only working on start.

Regards,

**Comment by tt4g** [comment]
There is no such API.
Please delete unnecessary log files manually by yourself.
